src/Test/Utilities/Portable/Compilation/CompilationExtensions.cs

```diff
@@ -255,6 +257,7 @@ public static void ValidateIOperations(Func<Compilation> createCompilation)
         {
 #if TEST_IOPERATION_INTERFACE
```

Comment:
Not related to this PR: why do we need this pragma?

Reply:
> Not related to this PR: why do we need this pragma?

So that we don't run this costly verification by default.